src/commands/api-mesh/create.js

```diff
@@ -150,7 +150,13 @@ class CreateCommand extends Command {
 								adobeIdIntegrationsForWorkspace.apiKey,
 							);
 
-							const { meshURL } = await getMesh(imsOrgId, projectId, workspaceId, mesh.meshId);
+							const { meshURL } = await getMesh(
+								imsOrgId,
+								projectId,
+								workspaceId,
+								workspaceName,
+								mesh.meshId,
+							);
 							const meshUrl =
```

Comment:
same here, this check needs to go away

Reply:
Removed the not needed checks. Now in case of nonTI users TI deligator will not return anything and CLI will check and append the api_key for nonTI users.